teo:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50 

  # Position Controllers ---------------------------------------
  waist_yaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: waist_yaw
    pid: {p: 100.0, i: 0.01, d: 10.0}
  waist_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: waist_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_hip_yaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_hip_yaw
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_hip_roll_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_hip_roll
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_hip_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_hip_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_knee_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_knee_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_ankle_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_ankle_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_ankle_roll_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_ankle_roll
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_hip_yaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_hip_yaw
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_hip_roll_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_hip_roll
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_hip_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_hip_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_knee_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_knee_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_ankle_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_ankle_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_ankle_roll_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_ankle_roll
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_shoulder_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_shoulder_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_shoulder_roll_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_shoulder_roll
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_shoulder_yaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_shoulder_yaw
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_elbow_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_elbow_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_wrist_yaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_wrist_yaw
    pid: {p: 100.0, i: 0.01, d: 10.0}
  r_wrist_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: r_wrist_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_shoulder_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_shoulder_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_shoulder_roll_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_shoulder_roll
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_shoulder_yaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_shoulder_yaw
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_elbow_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_elbow_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_wrist_yaw_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_wrist_yaw
    pid: {p: 100.0, i: 0.01, d: 10.0}
  l_wrist_pitch_position_controller:
    type: effort_controllers/JointPositionController
    joint: l_wrist_pitch
    pid: {p: 100.0, i: 0.01, d: 10.0}
